st2api/st2api/controllers/v1/keyvalue.py

```diff
@@ -97,7 +97,17 @@ def get_one(self, name, requester_user, scope=None, user=None, decrypt=False):
         key_ref = get_key_reference(scope=scope, name=name, user=user)
         extra = {"scope": scope, "name": name, "user": user, "key_ref": key_ref}
         LOG.debug("GET /v1/keys/%s", name, extra=extra)
-
+        LOG.audit(
```

Comment:
This should only be logged `if decrypt:` right? You can get keys without decrypting them.

Reply:
Hi @cognifloyd , 
Yes thats right just added the if condition.